packages/nextjs/src/config/webpack.ts

```diff
@@ -152,14 +149,14 @@ async function addSentryToEntryProperty(
   // because that then forces the user into a particular TS config.)
 
   // On the server, create a separate bundle, as there's no one entry point depended on by all the others
-  if (isServer) {
+  if (buildContext.isServer) {
     // slice off the final `.js` since webpack is going to add it back in for us, and we don't want to end up with
     // `.js.js` as the extension
-    newEntryProperty[SERVER_SDK_INIT_PATH.slice(0, -3)] = SENTRY_SERVER_CONFIG_FILE;
+    newEntryProperty[SERVER_SDK_INIT_PATH.slice(0, -3)] = SERVER_SDK_CONFIG_FILE;
```

Comment:
Good change

Reply:
The irony being that I made that change so it would match `SERVER_SDK_INIT_PATH`, and `SERVER_SDK_INIT_PATH` is about to be blown up... I still like the new name better, though, so will keep the change.